docs/developer/writing-exporter-flow-components.md

```diff
@@ -0,0 +1,55 @@
+# Create Prometheus Exporter Flow Components
```

Comment:
I linked process exporter and blackbox exporter somewhere in the middle of the docs, is that enough of you have something else in mind?

Reply:
Ah, sorry, my eyes glazed over those when scanning through the Markdown. Having a dedicated bullet list for some example/reference components would be nice, so that people looking just for links to examples don't have to read everything to find the links.